### PR TITLE
SUBMARINE-608. [WEB] Create environment through UI

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Complete creation environment through UI.

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-608

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/727321947

### Screenshots (if appropriate)
螢幕錄製 2020-09-15 下午4

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@pingsutw  @xunliu  @jiwq  Please help me review this pr. Thanks !

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/environment/environment.component.ts [hunk 1]
```diff
@@ -50,6 +59,100 @@ export class EnvironmentComponent implements OnInit {
...
+
+  addDependencies() {
+    this.dependencies.push(new FormControl('', Validators.required));
+  }
+
+  deleteItem(arr: FormArray, index: number) {
+    arr.removeAt(index);
+  }
+
+  submit() {
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/environment/environment.component.ts:117]:
```suggestion
  createEnvironment() {
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/environment/environment.component.ts [hunk 2]
```diff
@@ -50,6 +59,100 @@ export class EnvironmentComponent implements OnInit {
...
+
+    for (const dependency of this.dependencies.controls) {
+      environmentSpec.kernelSpec.dependencies.push(dependency.value);
+    }
+
+    return environmentSpec;
+  }
+
   // TODO(kobe860219): Create new environment
   createEnvironment(data) {}
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/environment/environment.component.ts:158]:
```suggestion
```

**Review by pingsutw — COMMENTED** [review]
Thanks @kobe860219 for the contribution

**Review by pingsutw — APPROVED** [review]
+1

**Review by jiwq — APPROVED** [review]
+1 Will commit if no more comments.

**Review by xunliu — COMMENTED** [review]
LGTM

**Pull request closed without merging**